### PR TITLE
HID UI updates [WIP]

### PR DESCRIPTION
**Pull request by jamesadevine**

### CODE REVIEW

**Review by pelikhan — APPROVED** [review]
Looks good. Make sure you have prettier installed and you format the file.

File: src/components/ui/KeyboardKeyInput.tsx [hunk 1]
```diff
@@ -221,12 +237,16 @@ export function renderKey(selector: number, modifiers: HidKeyboardModifiers) {
...
+            if (!convertToText)
+                values.push(flagText)
+            else 
+                values.push(`${display[flagText]}`)
         }
     })
     const sel = reverseSelectors[selector]
     if (sel !== undefined) values.push(sel.length > 1 ? `{${sel}}` : sel)
-    const value = values.filter(v => !!v).join(" ")
+    const value = values.filter(v => !!v).join(" + ")
```
Comment (pelikhan) [inline src/components/ui/KeyboardKeyInput.tsx:249]: this breaks simple-keyboard

**Review by pelikhan — CHANGES_REQUESTED** [review]
When generating the list of characters to highlight keys, don't add +

**Comment by jamesadevine** [comment]
Good catch! Thanks, will patch and use the convertToText flag to add the correct separator

**Comment by pelikhan** [comment]
I've fixed this function in another PR.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Pull request closed without merging**